services/web/server/src/simcore_service_webserver/projects/_metadata_api.py

```diff
@@ -19,12 +20,19 @@ async def get_project_metadata(
 
 
 async def set_project_custom_metadata(
```

Comment:
So this is not anymore "custom metadata".  The latter refer to metadata customized by the user . In the case of `parent_*_uuid` is more of structural or system metadata. . Perhaps we should rename these functions as `set_project_metadata`

Reply:
agree. was anyway weird